gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsInstrumentation.java

```diff
@@ -318,6 +318,18 @@ private void incrementMutableCounter(final String name, final long count) {
     }
   }
 
+  /**
+   * increment the failure statistics
+   *
+   * @param name
+   * @param count
+   */
+  public void incrementFailureStatistics(GhfsStatistic name, long count) {
+    if (count > 0) {
+      incrementMutableCounter(name.getSymbol() + SUFFIX_FAILURES, count);
+    }
+  }
+
```

Comment:
I don't think this is used anymore, we can remove this method

Reply:
Removed the method